misc/sampleconfigs/mpd.conf.sample

```diff
@@ -227,14 +255,14 @@ audio_output {
 #	device		"/dev/dsp"	# optional
 #	mixer_type      "hardware"      # optional
 #	mixer_device	"/dev/mixer"	# optional
-#	mixer_control	"%AUDIOiFace%"		# optional
+#	mixer_control	"PCM"		# optional
```

Comment:
is this intended?

Reply:
Thats a display problem due to the renaming and not a real change.
The obsolete and outdated mpd.conf file that was removed before, had also the name "mpd.conf.sample".
In the overall change view this is now display as changes, rather deletion and renaming.

If you just view the last commit you can see no changes of content and just the renaming of the file.